tests/dummy/app/components/demo-page.js

```diff
@@ -0,0 +1,30 @@
+import Component from '@ember/component';
+import $ from 'jquery';
+
+const DEVILS_NUMBER = 666 / 2; // Give this a proper name  (╯°□°）╯︵ ┻━┻
+
+export default Component.extend({
+
+  didRender() {
+    this._super(...arguments);
+
+    $(document).on('click', 'a[href^="#"]', function(event) {
```

Comment:
lets move all these selectors to a options object? 😛

Reply:
☝️